pkgs/development/tools/codeowners/default.nix

```
@@ -0,0 +1,23 @@
+{ buildGoModule, lib, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "codeowners";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "hmarr";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-YhGBg7CP5usSyP3ksX3/54M9gCokK2No/fYANUTdJw0=";
+  };
+
+  vendorSha256 = "sha256-no1x+g5MThhWw4eTfP33zoY8TyUtkt60FKsV2hTnYUU=";
+
+  meta = with lib; {
+    description = "A CLI and Go library for Github's CODEOWNERS file";
+    license = licenses.mit;
+    maintainers = with maintainers; [ yorickvp ];
+    inherit (src.meta) homepage;
```

Much easier to have the actual URL. Otherwise it requires extra effort for people reviewing updates in the futures to get to upstream's website. 

```suggestion
    homepage = "https://github.com/hmarr/codeowners";
```